test/ZendeskApi_v2.Test/TicketTests.cs

```diff
@@ -1119,6 +1119,84 @@ public void CanImportTicketAsync()
             api.Tickets.DeleteAsync(res.Id);
         }
 
+        [Test]
+        public void CanMergeTickets()
+        {
+            var sourceDescription = new List<string> { "This is a source ticket #1", "This is a source ticket #2" };
+            var targetDescription = "This is a the target ticket";
+
+            var sourceTicket1 = new Ticket
+            {
+                Subject = "Source Ticket 1",
+                Comment = new Comment { Body = sourceDescription[0], Public = true, }
+            };
+            var sourceTicket2 = new Ticket
+            {
+                Subject = "Source Ticket 2",
+                Comment = new Comment { Body = sourceDescription[1], Public=true, }
+            };
+            var targetTicket = new Ticket
+            {
+                Subject = "Target Ticket",
+                Comment = new Comment { Body = targetDescription, Public = true, }
+            };
+
+            var mergeIds = new List<long>();
+
+            var tick = api.Tickets.CreateTicket(sourceTicket1);
+            mergeIds.Add(tick.Ticket.Id.Value);
+            tick = api.Tickets.CreateTicket(sourceTicket2);
+            mergeIds.Add(tick.Ticket.Id.Value);
+            tick = api.Tickets.CreateTicket(targetTicket);
+            var targetTicketId = tick.Ticket.Id.Value;
+
+            var targetMergeComment = $"Merged with tickets {string.Join(", ", mergeIds.ToArray())}";
+            var sourceMergeComment = $"Closing in favor of {targetTicketId}";
+
+            var res = api.Tickets.MergeTickets(
+                targetTicketId, 
+                mergeIds, 
+                targetMergeComment, 
+                sourceMergeComment);
+
+            Assert.AreEqual(res.JobStatus.Status, "queued");
```

Comment:
```suggestion
            Assert.That(res.JobStatus.Status, Is.EqualTo("queued"));
```
please change all Asserts to use the ```Assert.That``` style.

Reply:
Changing... used .AreEqual based on your existing code. :)